### ISSUE TITLE
Wire not support Aws Hong Kong Region(ap-east-1)

### ISSUE THREAD

**Issue by limengyu1990** [wireapp/wire-server#955]
### gundeck.yaml  config info 
aws:
  queueName: im-gundeck-events
  region: ap-east-1
  account: "11111111" 
  arnEnv: test
  sqsEndpoint: https://sqs.ap-east-1.amazonaws.com
  snsEndpoint: https://sns.ap-southeast-1.amazonaws.com

### error info
gundeck: user error (AesonException "Error in $.aws.region: Failed reading: Failure parsing Region from ap-east-1")

**Comment by limengyu1990** [comment]
### error info(`core/src/Network/AWS/Types.hs`):
```haskell
user error (AesonException "Error in $.aws.region: Failed reading: Failure parsing Region from ap-east-1")
```

I noticed that `wire` used its own maintained` amazonka` branch and fixed 2 bugs (`important commits`).
I want to know if the `wire` team has time and plans to synchronize the`amazonka` branch of `wire` with the remote newest `amazonka` and then re-package an `archive` package? thanks.

### wire source code Comment
```haskell
# amazonka-1.6.0 is buggy: https://github.com/brendanhay/amazonka/issues/466
# amazonka-HEAD is also buggy: https://github.com/brendanhay/amazonka/issues/490
#
# Therefore we use our own fork of amazonka here. More precisely, we pull two libraries out of
# it: amazonka and amazonka-core. Other packages weren't changed between 1.6.0 and this
# commit, so we can use Stackage-supplied versions for them.
#
# The important commits for us are:
#
#   * https://github.com/brendanhay/amazonka/commit/2688190f
#   * https://github.com/brendanhay/amazonka/pull/493/files
#
# The archive corresponds to commit 52896fd46ef6812708e9e4d7456becc692698f6b.
- archive: https://github.com/wireapp/amazonka/archive/wire-2019-01-25.tar.gz
  subdirs:
  - amazonka
  - core
```


### amazonka
https://hackage.haskell.org/package/amazonka-1.6.1/docs/Control-Monad-Trans-AWS.html#t:Region
https://github.com/brendanhay/amazonka/blob/725340d92f8109bf5ea620a224fe613df9fba8f1/core/src/Network/AWS/Types.hs#Region

@brendanhay
https://github.com/wireapp/amazonka/archive/wire-2019-01-25.tar.gz

**Comment by mheinzel** [comment]
Hi!

Getting PR https://github.com/brendanhay/amazonka/pull/493 merged and released on Hackage is the only thing keeping us from switching back to the official amazonka releases. I rebased the PR and maybe one of the maintainers will have the time to look into it, but I'm not sure.

If nothing happens with the PR next week, I'll make a new release of our updated fork.

**Comment by mheinzel** [comment]
The fix got merged into amazonka and I will now merge #976, which will make us use the new version with support for the ap-east-1 region.
